doc/plugins/eu.esdihumboldt.hale.doc.user/html/introduction/introduction_dhp.html

```diff
@@ -2,37 +2,20 @@
 <html>
 <head>
 <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
-<title>The data harmonisation panel</title>
+<title>wetransform</title>
 </head>
 <body>
-	<h1>The data harmonisation panel</h1>
+	<h1>wetransform</h1>
 
-	<p>From April 2011 on, after the end of the HUMBOLDT project, the
-		development of <i>hale studio</i> has been coordinated under the auspices of the
-		data harmonisation panel. The data harmonisation panel has been
-		established with the goal of supporting the international community of
-		experts and organisations that have to deal with spatial data
-		harmonisation by disseminating, exploiting and advancing harmonisation
-		methods and technologies such as the tools and services developed in
-		the HUMBOLDT project.</p>
+	<p>In 2015, wetransform took over the responsibility to develop and support the open source ETL tool <i>hale»studio</i>. Before that, the development was mostly financed through EU projects such as HUMBOLDT, which enabled rapid development.
+		Since wetransform took over, we’ve been able to support thousands of users in using <i>hale»studio</i> to harmonise their data and to comply with many national and international standards, be it for e-Reporting, INSPIRE, CityGML, NAS or XPlanGML.
+		Also, we continue to be pleasantly surprised by the kind of problems people solve using <i>hale»studio</i>.</p>
 
-	<p>Join the data harmonisation panel in these activities:</p>
-	<ul>
-		<li>Advancement of the cooperation among the stakeholders of
-			geo-data harmonisation, including standardisation efforts</li>
-		<li>Research of current developments and trends in the data
-			harmonisation field, including market analysis</li>
-		<li>Continued maintenance, development, and exploitation of
-			state-of-the-art tools and services of HUMBOLDT</li>
-		<li>Support of the development of the European Spatial Data
-			Infrastructure as planned by INSPIRE</li>
-		<li>Transfer of expertise into different application domains such
-			as Security, Environment and Disaster Management.</li>
-	</ul>
+	<p>wetransform is a Fraunhofer spin-off established in 2015. Our mission is to "Make Open Standards work" for data exchange.
+		Through the unique <i>hale»studio</i> data harmonisation technology and our <i>hale»connect</i> cloud platform, we make spatial and environmental data accessible and usable and thus provide the foundation for data ecosystems around topics such as Environmental Reporting, UAV Flight Zone Management and Forest Transformation.</p>
 
-	<p>
-		For more information on the data harmonisation panel please visit <a
-			href="http://www.dhpanel.eu" target="_blank">www.dhpanel.eu</a>.
-	</p>
+	<p>We offer a wide range of services around data harmonisation for national and international standards and cover the whole process from transformation to publishing and quality assurance. Find out more on our website at <a
```

Comment:
Is it expected that there is a line break within the HTML element?

Reply:
Yes.